views/backend/settings.php

```diff
@@ -42,11 +42,19 @@
 
         <?php
             if(isset($_POST['submit'])){
-
-                $name = htmlspecialchars(trim($_POST['name']));
-                $email = htmlspecialchars(trim($_POST['email']));
-                $email_again = htmlspecialchars(trim($_POST['email_again']));
-                $role = htmlspecialchars(trim($_POST['role']));
+                if(isset($_POST['name'])){
+                    $name = htmlspecialchars(trim($_POST['name']));
+                }
+                if(isset($_POST['email'])){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)